src/components/Home/Meetings/views/__tests__/Instant.test.tsx

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { Instant } from "../Instant";
+
+describe("Instant", () => {
+  test.todo("render");
```

Comment:
I can only approve the PR if the tests actually work

Reply:
These are not critical tests. I'll write them after implementing the main functionality in [this story](https://app.shortcut.com/deskpro/story/106868/zoom-improvements)

Reply:
done